frontend/src/metabase/nav/components/ProfileLink.jsx

```diff
@@ -57,9 +57,7 @@ export default class ProfileLink extends Component {
       {
         title: t`Help`,
         icon: null,
-        // HACK - for some reason if you use // react router treats the link
-        // as a non local route
-        link: `//metabase.com/docs/${tag}`,
+        link: `https://metabase.com/docs/${tag}/`,
```

Comment:
@tlrobinson is the right way to make the link always HTTPS?

Reply:
@camsaul 
By using `//metabase.com`, you're telling the browser to visit the link, but keep using the same protocol as current site.
So, if you're sitting on `http://metabase.example.com`, then it will continue to use `http` as link.

Reply:
right, just asking @tlrobinson if that works correctly because of the `HACK` comment there.